server/rak.js

```diff
@@ -9,11 +9,15 @@ const buttonAlerts = require('./buttonAlerts')
 const rakApiKeys = [helpers.getEnvVar('RAK_API_KEY_PRIMARY'), helpers.getEnvVar('RAK_API_KEY_SECONDARY')]
 
 const EVENT_TYPE = {
-  BUTTON_PRESS_1: 65, // ASCII for 'A'
-  BUTTON_PRESS_2: 66, // ASCII for 'B'
-  BUTTON_PRESS_3: 67, // ASCII for 'C'
-  BUTTON_PRESS_4: 68, // ASCII for 'D'
+  BUTTON_PRESS_A: 65, // ASCII for 'A'
```

Comment:
Maybe re-order these so the ASCII values are ordered; the ordering would then be like:

BUTTON_PRESS_1,
BUTTON_PRESS_2,
BUTTON_PRESS_3,
BUTTON_PRESS_4,
BUTTON_PRESS_A,
BUTTON_PRESS_B,
BUTTON_PRESS_C,
BUTTON_PRESS_D,
HEARTBEAT

Reply:
Also good that you renamed the old values so the BUTTON_PRESS_X corresponds to what ASCII character it references!

Reply:
Woooo, that was Johnny's suggestion!